war/src/main/js/add-item.js

```diff
@@ -276,6 +284,29 @@ $.when(getItems(jRoot)).done(function(data){
       return $item;
     }
     
+    function drawIcon(elem){
+      var $icn = $('<div class="icn">');
+      if (!elem.iconFilePathPattern) {
+        var name = elem.displayName;
+        var aName = name.split(' ');
+        var a = name.substring(0,1);
+        var b = ((aName.length === 1)?
+            name.substring(1,2):
+              aName[1].substring(0,1));
+        var $defIcon = $([
```

Comment:
I often add line breaks on inline conditionals so I can drop a debugger on the condition, if necessary, but will 1 line it, if you prefer.